subresource-loading.bs

```diff
@@ -180,9 +239,10 @@ To <dfn id="concept-fetch-from-web-bundle">fetch from web bundle</dfn> given
 1. For each |web bundle| of |httpRequest|'s [=request/client=]'s [=environment
    settings object/web bundle list=]:
 
-   1. If the result of running [=match url with web bundle=] given
-      |httpRequest|'s [=request/url=] and |web bundle|'s [=web bundle/rule=] is
-      true:
+   1. Let |web bundle| be the result of running [=find matching web bundle=]
```

Comment:
It seems we can remove the outer loop (L239), which is no longer required.

Reply:
Good catch, done.